src/modules/saveReminder/SaveReminder.ts

```diff
@@ -0,0 +1,60 @@
+import {
+    Observable as KnockoutObservable,
+} from 'knockout';
+import { Saveable } from '../DataStore/common/Saveable';
+import * as GameConstants from '../GameConstants';
+import Notifier from '../notifications/Notifier';
+import Settings from '../settings';
+
+export default class SaveReminder implements Saveable {
+    public static counter = 0;
+
+    saveKey = 'saveReminder';
+
+    defaults: Record<string, any> = {};
+
+    public lastDownloaded: KnockoutObservable<number> = ko.observable(0).extend({ numeric: 0 });
+    public lastReminder: KnockoutObservable<number> = ko.observable(0).extend({ numeric: 0 });
+
+    // eslint-disable-next-line @typescript-eslint/no-useless-constructor, @typescript-eslint/no-empty-function
+    constructor() {}
+
+    static tick() {
+        this.counter = 0;
+
+        // Check if reminders are disabled
+        if (!+Settings.getSetting('saveReminder').value) return;
+
+        let timeSinceSaveReminder = App.game.statistics.secondsPlayed() - Math.max(App.game.saveReminder.lastDownloaded(), App.game.saveReminder.lastReminder());
+        // Adjust to measure against ms
+        timeSinceSaveReminder *= 1000;
+
+        if (timeSinceSaveReminder >= +Settings.getSetting('saveReminder').value) {
+            // Show reminder notification
+            Notifier.notify({
+                title: 'Save Reminder',
+                message: `It has been ${GameConstants.formatTimeShortWords(+Settings.getSetting('saveReminder').value)} since your last save download, Would you like to download a backup now?
+
+                <button class="btn btn-block btn-success" onclick="Save.download()" data-dismiss="toast">Download Save</button>`,
+                timeout: 3 * GameConstants.HOUR,
```

Comment:
Because the lowest value is 1 hour, the player can end up with 2 popups on their screen, which some might find annoying

Reply:
Good point 👍
Changed it to be whichever is lower (the interval or 3 hours)